web/client/selectors/mapInfo.js

```diff
@@ -72,13 +70,11 @@ const stopGetFeatureInfoSelector = createSelector(
     measureActiveSelector,
     drawSupportActiveSelector,
     annotationsEditingSelector,
-    queryPanelSelector,
-    (isMapInfoDisabled, isMeasureActive, isDrawSupportActive, isAnnotationsEditing, isQueryPanelActive) =>
+    (isMapInfoDisabled, isMeasureActive, isDrawSupportActive, isAnnotationsEditing) =>
```

Comment:
Please check if tests needs to be updated, especially if all cases of stopGetFeatureInfoSelector are covered

Reply:
It seems all tests are fine and the change does not negatively affect anything.

Reply:
ok